src/config.ts

```diff
@@ -927,8 +927,8 @@ export default {
    * @type {Object}
    */
   ADAPTATION_SWITCH_BUFFER_PADDINGS: {
-    video: { before: 0.5, after: 1 },
-    audio: { before: 0.5, after: 2 },
+    video: { before: 1, after: 1 },
```

Comment:
Nice, i find it better to let at least 2 secondes of buffered data around current time. We could even test to let 3/4 secondes, to higher the chances to cover an entire chunk (or necessary I image to decode current time ?)

Reply:
The problem with 3 / 4 seconds is that it may postpone the time at which an Adaptation will be switched to.
OK to put it at something like 2 before / 2.5 after

Reply:
Done